crates/katana/src/args.rs

```diff
@@ -148,7 +149,8 @@ pub struct EnvironmentOptions {
     #[arg(long)]
     #[arg(help = "The chain ID.")]
     #[arg(default_value = "KATANA")]
-    pub chain_id: String,
+    #[arg(value_parser = ChainId::parse)]
```

Comment:
We could add a long help with the comment you have in the code about hex string parsing, or a cairo short string otherwise.

Reply:
good idea